pkg/controller/pxc/volumes.go

```diff
@@ -120,12 +126,15 @@ func (r *ReconcilePerconaXtraDBCluster) reconcilePersistentVolumes(ctx context.C
 
 	configured := volumeTemplate.Spec.Resources.Requests[corev1.ResourceStorage]
 	requested := cr.Spec.PXC.VolumeSpec.PersistentVolumeClaim.Resources.Requests[corev1.ResourceStorage]
+	gib, err := RoundUpGiB(requested.Value())
+	if err != nil {
+		return errors.Wrap(err, "round GiB value")
+	}
 
-	if requested.Format == resource.DecimalSI {
-		requested, err = resource.ParseQuantity(requested.String() + "i")
-		if err != nil {
-			return errors.Wrap(err, "parse requested storage size")
-		}
+	requestedQuantity := fmt.Sprintf("%dGi", gib)
+	requested, err = resource.ParseQuantity(requestedQuantity)
+	if err != nil {
+		return errors.Wrapf(err, "parse quantity (%s)", requestedQuantity)
 	}
```

Comment:
I'm fine with this implementation, but what do you think of this one:

```suggestion
	requested = resource.NewQuantity(gib * GiB, resource.BinarySI)
```